dotCMS/src/test/java/com/dotcms/mock/MockRequestTest.java

```diff
@@ -82,4 +82,21 @@ public void testMockInternalRequest() {
 		assertThat("request isSecure is false", !request.isSecure());
 		assertThat("request is a httprequest ", request instanceof HttpServletRequest);
 	}
+	
+	
+    @Test
+    public void testMockHttpRequestWithNullValues() {
+        final String expectedHostname="localhost";
+        final String expectedUri="/";
+        HttpServletRequest request =    new MockHttpRequest(null,null).request();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'request' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5505198)